src/main/java/es/urjc/cloudapps/library/data/BookDto.java

```diff
@@ -0,0 +1,21 @@
+package es.urjc.cloudapps.library.data;
+
+public class BookDto {
+
+    private String uuid;
```

Comment:
Como ves usar 'id'? Asi el cliente no se acopla a si es un UUID, un base64...

```suggestion
    private String id;
```